tkweb/apps/drinks/migrations/0002_barcard.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-10-28 17:45
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('drinks', '0001_initial'),
```

Comment:
Der er ikke nogen grund til at have to migrations - de bør slås sammen til én i dette pull request ved at slette de to migrations-filer og køre `makemigrations` igen. Det simpleste er at slette databasen i dit udviklingsmiljø `db.sqlite3`, men hvis du vil gøre det i dit udviklingsmiljø uden at miste det data du har lavet i drinks-appen lokalt, kan du gøre som følger:

1. `./manage.py migrate --fake drinks zero`
2. `rm tkweb/apps/drinks/migrations/00*.py`
3. `./manage.py makemigrations drinks`
4. `./manage.py migrate --fake drinks`
5. `git add tkweb/apps/drinks/migrations`

Reply:
Bør kommandoerne køres i `pipenv shell`?

Reply:
hmm... når jeg køre den første kommando får jeg følgende fejl:

    File "/home/fuan/.local/share/virtualenvs/web-22tkNSwG/lib/python3.5/site-packages/django/urls/resolvers.py", line 416, in url_patterns
        raise ImproperlyConfigured(msg.format(name=self.urlconf_name))
    django.core.exceptions.ImproperlyConfigured: The included URLconf 'tkweb.urls' does not appear to have any patterns in it. If you see valid patterns in the file then the issue is probably caused by a circular import.

Reply:
Det er vidst fordi jeg har lavet noget rod i koden og ikke specifikt til komandoen. Fikser det lige, eller skriver jeg.